WalletWasabi.Fluent/Models/Wallets/WalletListModel.cs

```diff
@@ -11,6 +11,8 @@ namespace WalletWasabi.Fluent.Models.Wallets;
 
 public partial class WalletListModel : ReactiveObject, IWalletListModel
 {
+	[AutoNotify] private IWalletModel? _selectedWallet;
```

Comment:
Are we planning to use this in the future? For me it seems a bit overkill to have a separate property and keep the selected wallet synchronized on model level too. (btw now it only works from VM -> Model, but not from Model -> VW, which could cause confusion)

A simple method on model level shouldn't be enough? `StoreLastSelectedWallet()`

Reply:
Changed to method.